cinema/serializers.py

```diff
@@ -135,3 +137,40 @@ def create(self, validated_data):
 
 class OrderListSerializer(OrderSerializer):
     tickets = TicketListSerializer(many=True, read_only=True)
+
+
+class UserRegisterSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=5)
```

Comment:
The `min_length` for the password is set to 5. It's generally recommended to have a longer minimum password length for better security, typically at least 8 characters.